### PR TITLE
Use type strategy with deposit

### PR DESCRIPTION
**Pull request by lwrubel**
## Why was this change made?
Follow on to #178 to be able to use sdr-client to deposit files that are images. 
 
## How was this change tested?

Running specs, adding a test.

### CODE REVIEW

**Comment by lwrubel** [comment]
I can squash commits once this is approved, unless best practice is to squash first.

**Comment by mjgiarlo** [comment]
@lwrubel 💬 
> I can squash commits once this is approved, unless best practice is to squash first.

I think it's A-OK to say "pls squash these commits, person-who-merges?"

**Review by mjgiarlo — APPROVED** [review]

**Comment by mjgiarlo** [comment]
Squash-merged. Thanks, @lwrubel!

**Pull request merged**
